deploy/airgap-prepare.sh

```diff
@@ -33,19 +33,19 @@ fi
 # Download k3s
 if [[ ! -f $K3S_BINARY ]]
 then
-	curl -kL -o $K3S_BINARY  https://github.com/rancher/k3s/releases/download/v1.18.10%2Bk3s1/k3s
```

Comment:
Are there e2e tests to verify that going from 1.18.10 to 1.23.6 during RPM upgrade will be sucessful? (ex: upgrade from auth 1.2 to 1.3)